webapp/src/desktop_notifications.ts

```diff
@@ -21,6 +33,20 @@ export function desktopNotificationHandler(store: Store, post: Post, channel: Ch
             ringingEnabled(store.getState())) {
             return {args: {...args, notify: false}};
         }
+
+        // Do not notify for a call's thread if the user is currently in that call...
+        if (channelIDForCurrentCall(store.getState()) === post.channel_id &&
+            callThreadIDForCurrentCall(store.getState()) === post.root_id) {
+            let mentions = [];
+            if (msgProps.mentions) {
+                mentions = JSON.parse(msgProps.mentions);
+            }
+
+            // ...and wasn't directly mentioned.
```

Comment:
Nice, I wasn't sure myself if we wanted this exception but it probably makes sense.